### DISCUSSION TITLE
adding session-token to kvssink

### DISCUSSION THREAD

**Discussion by clogwog** [awslabs/amazon-kinesis-video-streams-producer-sdk-cpp#989] [Ideas]
We have a use case where we retrieve temporary AWS credentials directly from greengrass and it comes in the form of

access-key
secret-key
session-token

currently kvssink only picks up the AWS_SESSION_TOKEN (from the environment variable ) if the AWS_ACCESS_KEY and AWS_SECRET_KEY are set as environment variables. Never from the kvssink parameters.

I've locally made a small change to allow an extra session-token to be picked up from the kvssink parameters if access-key and 
secret-key are also defined and it seems to work fine for us.

Is there a reason why this currently is not in this repository ? and if there isn't shall i try a pull request for this ?
kind regards,
tom

**Comment by disa6302** [comment]
@clogwog ,

It was a missing property, but should be included soon. I have a PR out that covers this property. 
https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp/pull/957

**Reply by clogwog** [reply]
That is awesome!! Thanks
